zap/src/main/java/org/zaproxy/zap/eventBus/SimpleEventBus.java

```diff
@@ -19,16 +19,14 @@
  */
 package org.zaproxy.zap.eventBus;
 
+import static java.util.Arrays.asList;
+
 import java.security.InvalidParameterException;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Map;
-import java.util.Map.Entry;
+import java.util.*;
```

Comment:
Please use specific imports not wildcard.

Reply:
Argh, I blame IDEA

Reply:
No worries. Intelli-J tends to do that on default settings as well.